### ISSUE TITLE
Sync slow to never?

### ISSUE THREAD

**Issue by combinatorist** [padlock/padlock#191]
## Problem
I have noticed that when I add new entries or fields in existing entries on my **Mac Book Pro**, they do not show up on other devices (**Mac Air and iPhone 5**), despite doing all the following:
- manually clicking the sync button on the Pro, and then on the other device
- waiting several minutes and doing the same
- verifying under the cloud settings that all my devices are listed (same account)
- verifying that my subscription is paid

## Possible Causes
- My MacBook Pro is a work computer with lots of security junk on it.
- My payment method was out of date recently (card expired), but how long should it take to stop syncing to cloud and how long to start again?

## Expected Behavior
If my Pro was not able or allowed to sync to my cloud account, I would expect an error message, so I know to jot my password on a post-it ( 😮). Similarly, if my other devices aren't able to bring the update down from the cloud, I'd expect an error. Otherwise, I'm left with a false sense of security that my sensitive data is backed up ... followed by crippling anxiety that it isn't. 😉

**Comment by MaKleSoft** [comment]
Can't reproduce this on my side so I'm guessing it's got something to do with your local setup on your mac. Security software like firewalls can definitely effect the apps ability to access the network. You've mentioned that you're seeing the delay specifically when syncing with your Macbook Pro? Maybe try to sync your iPhone and Macbook Air and see if you're getting the same problem. If you send me an email with your account address I'll also be happy to check if there is any problem with your subscription.

**Comment by MaKleSoft** [comment]
Closing this for now since the problem described seems to be due to local factors which are out of the apps control.

**Comment by combinatorist** [comment]
FWIW, I just retested from MacBookPro to iPhone 5 and vice versa:
- When I made a change, the source device synced automatically.
- If the target device was already open before the change, it wouldn't sync (waited 2 minutes).
- However, if I manually synced, it updated within a second.

It seems fixed and I agree - I don't think this was an issue with the software. :+1 It was probably some issues with work configurations on the laptop or maybe my temporarily delinquent account. :)
